maintainers/gpuci.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: default
+nav_order: 4
+parent: RAPIDS Maintainer Docs
+title: gpuCI
+---
+
+# gpuCI Technical Documentation
+
+## Overview
+
+Outlines how gpuCI is configured for RAPIDS and how to integrate other projects
+
+gpuCI is accessible here: https://gpuci.gpuopenanalytics.com/
+
+### Intended audience
+
+Developers
+{: .label .label-green}
+
+Project Leads
+{: .label .label-blue}
+
+Operations
+{: .label .label-purple}
+
+## Infrastructure
+
+This is an overview of the tools used in gpuCI
+
+### Jenkins
+
+- Open source project for CI/CD and general automation that is very extensible and well adopted
+- gpuCI
```

Comment:
I'm a little confused about the bullet list that follows and how it's related to Jenkins. It seems like it should be its own subsection? I see that this was taken from the older version of `testing.md`, but maybe with the new style used here in this doc (paragraphs with full sentences), the bullet list doesn't fit anymore?